apps/mobile-wallet/app/[...missing].tsx

```diff
@@ -0,0 +1,41 @@
+import { StyleSheet } from 'react-native';
+
+import { Link, Stack } from 'expo-router';
+
+import { Text, View } from '../components/Themed';
+
+export default function NotFoundScreen() {
+  return (
+    <>
+      <Stack.Screen options={{ title: 'Oops!' }} />
+      <View style={styles.container}>
+        <Text style={styles.title}>This screen doesn't exist.</Text>
+
+        <Link href="/" style={styles.link}>
+          <Text style={styles.linkText}>Go to home screen!</Text>
+        </Link>
+      </View>
+    </>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 20,
+  },
+  title: {
+    fontSize: 20,
+    fontWeight: 'bold',
+  },
+  link: {
+    marginTop: 15,
+    paddingVertical: 15,
+  },
+  linkText: {
+    fontSize: 14,
+    color: '#2e78b7',
```

Comment:
Maybe we can have tokens for colours from the start? (not a big deal for now / this PR)

Reply:
I can address in a separate PR, i expect adding the tokens for colors, typography, sizes, etc. might be a bit too much for this PR.

Reply:
makes sense, just mentioning now 👍